src/Elastic.Apm/Helpers/WildcardMatcher.cs

```diff
@@ -110,7 +110,7 @@ public static WildcardMatcher ValueOf(string wildcardString)
 		/// <returns>The first matching <see cref="WildcardMatcher" />, or <code>null</code> if none match.</returns>
 		internal static WildcardMatcher AnyMatch(IReadOnlyCollection<WildcardMatcher> matchers, string firstPart, string secondPart)
 		{
-			for (var i = 0; i < matchers.Count; i++)
+			for (var i = 0; i < matchers?.Count; i++)
```

Comment:
`if (matchers is null) return null;` on the first line might be easier to read. 

Had to have a look at the IL to see what this results in, and it instantiates an `int?` and checks if it has a value initialized from a non-null matcher count on each loop, so the initial null check looks like it'd be better.

Reply:
Ok - changed this to the suggested null check.